src/monitord/monitor_actions.c

```diff
@@ -50,7 +50,8 @@ void monitor_agents_disconnection(){
     int *agents_array;
     char str_agent_id[12];
 
-    agents_array = wdb_disconnect_agents(time(0) - mond.global.agents_disconnection_time, &sock);
+    agents_array = wdb_disconnect_agents(time(0) - mond.global.agents_disconnection_time,
+                                         !mond.monitor_agents?"syncreq":"synced", &sock);
```

Comment:
It may happen that an user turns off the _monitor_agents_ option in a master node. Would it be a problem that the disconnected agents have a **syncreq** status in that case?

Reply:
It's true. It's not usual but can be used in that way.

Reply:
Good catch guys! Fixed!